src/mrdsrg-spin-integrated/dwms_mrpt2.cc

```diff
@@ -417,17 +416,16 @@ DWMS_DSRGPT2::compute_macro_dsrg_pt(std::shared_ptr<MASTER_DSRG>& dsrg_pt,
     print_sa_info("Original State Averaging Summary", sa_info);
     print_sa_info("Reweighted State Averaging Summary", sa_info_new);
 
-    // compute Reference
+    // compute RDMs
     fci_mo->set_sa_info(sa_info_new);
-    fci_mo->set_max_rdm_level(max_rdm_level_);
-    Reference reference = fci_mo->reference();
+    RDMs rdms = fci_mo->reference(max_rdm_level_); // TODO: bug here? (Francesco)
 
     // update MK vacuum energy
-    //double new_Eref = compute_Eref_from_reference(reference, ints_, mo_space_info_, Enuc_);
-    //reference.set_Eref(new_Eref); // TODO: ?why do this here this way?
+    // double new_Eref = compute_Eref_from_rdms(reference, ints_, mo_space_info_, Enuc_);
+    // rdms.set_Eref(new_Eref); // TODO: ?why do this here this way?
```

Comment:
Because the MK vacuum changed. WIthout this update will trigger the warning of inconsistent reference energy in DSRG.

Reply:
This does not matter now.